Tests/Rules/EnvironmentEntryTests.swift

```diff
@@ -0,0 +1,170 @@
+// Created by miguel_jimenez on 10/16/24.
+// Copyright © 2024 Airbnb Inc. All rights reserved.
+
+import XCTest
+
+final class EnvironmentEntryTests: XCTestCase {
+    func testReplaceEnvironmentKeyDefinitionForEntryMacro() {
+        let input = """
+        struct ScreenNameEnvironmentKey: EnvironmentKey {
+            static var defaultValue: Identifier? {
+                .init("undefined")
+            }
+        }
+
+        extension EnvironmentValues {
+            var screenName: Identifier? {
+                get { self[ScreenNameEnvironmentKey.self] }
+                set { self[ScreenNameEnvironmentKey.self] = newValue }
+            }
+        }
+        """
+        let output = """
+        extension EnvironmentValues {
+            @Entry var screenName: Identifier? = .init("undefined")
+        }
+        """
+        testFormatting(for: input, output, rule: .environmentEntry)
+    }
+
+    func testReplaceEnvironmentKeyDefinitionForEntryMacroWithKeyDefinitionAfterEnvironmentValue() {
+        let input = """
+        extension EnvironmentValues {
+            var screenName: Identifier? {
+                get { self[ScreenNameEnvironmentKey.self] }
+                set { self[ScreenNameEnvironmentKey.self] = newValue }
+            }
+        }
+
+        struct ScreenNameEnvironmentKey: EnvironmentKey {
+            static var defaultValue: Identifier? {
+                .init("undefined")
+            }
+        }
+        """
+        let output = """
+        extension EnvironmentValues {
+            @Entry var screenName: Identifier? = .init("undefined")
+        }
+
+        """
+        testFormatting(
+            for: input, [output],
+            rules: [
+                .environmentEntry,
+                .blankLinesBetweenScopes,
+                .blankLinesAtEndOfScope,
+                .blankLinesAtStartOfScope,
+                .consecutiveBlankLines,
+            ]
+        )
+    }
+
+    func testReplaceMultipleEnvironmentKeyDefinitionForEntryMacro() {
+        let input = """
+        extension EnvironmentValues {
+            var isSelected: Bool {
+                get { self[IsSelectedEnvironmentKey.self] }
+                set { self[IsSelectedEnvironmentKey.self] = newValue }
+            }
+        }
+
+        struct IsSelectedEnvironmentKey: EnvironmentKey {
+            static var defaultValue: Bool { false }
+        }
+
+        extension EnvironmentValues {
+            var screenName: Identifier? {
```

Comment:
Would be good to have a test case that include doc comments on the `EnvironmentValues` definition, to verify that those aren't accidentally removed or anything (It looks like they'd be preserved based on the existing rule implementation but it's always good to capture in a test)

```suggestion
            /// The name of this screen for logging purposes. Defaults to `"undefined"`.         
            var screenName: Identifier? {
```